CHANGES.rst

```diff
@@ -1,6 +1,20 @@
 Latest changes
 ===============
 
+Release 0.10.0
+--------------
+
+Alexandre Abadie
+
+    Joblib now uses a single file when pickling objects with dump and
```

Comment:
I would say something like:

Joblib pickle format change: joblib.dump always create a single pickle file and joblib.dump/joblib.save never do any memory copy when writing/reading pickle files. Reading pickle files generated with joblib versions prior to 0.10 will be supported for a limited amount of time, we advise to regenerate them from scratch when convenient.

Feel free to rephrase.